picard/ui/filedialog.py

```diff
@@ -0,0 +1,77 @@
+from picard.ui import PicardDialog
+from picard.ui.ui_filedialog import Ui_fileDialog
+from picard.formats import supported_extensions
+from PyQt5.QtWidgets import (
+        QFileSystemModel , QAbstractItemView ,
+        QListWidgetItem)
+from PyQt5.QtCore import QStorageInfo, QDir, QSortFilterProxyModel , QRegExp
+import os,sys
```

Comment:
Please import only one module at a time. Also order your imports. See https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Also see the codacy issues - https://www.codacy.com/app/MetaBrainz/picard/pullRequest?prid=1214235